installers/README.markdown

```diff
@@ -135,3 +136,13 @@ For more detailed information about the installation process and errors, you can
     $ sudo ./ubuntu.sh --debug
     $ su -c "./debian.sh --debug"
 
+## Installing on RHEL / CentOS 8
+
+Please note that the RHEL/CentOS 8 installer is currently in beta and there may be some issues with functionality.
+If trying to view the DreamFactory UI in the browser after installation leads to a bad gateway error / an nginx error page, please try the following:
+
+1. Go to `/etc/php-fpm.d` and on / around line 38 check (and change if necessary) the `listen` to `listen = 127.0.0.1:9000` and save.
```

Comment:
Could you give me more insights into why the 9000 port is used? and overall what is this change of fpm config?

Reply:
Sorry, should have been a bit more clear on that.
My understanding is that php-fpm listens on port 9000 to accept requests from nginx (i.e fastCGI requests), and if you look at the `/etc/php-fpm/www.conf` file (actually I need to edit what I wrote) on a RHEL 7 installation, it is set to `127.0.0.1:9000`

But on RHEL 8, for some reason the same file (php-fpm.d/www.conf) defaults to listening to these requests on a unix socket `/run/php-fpm/www.sock` , and that socket is incorrect (you wont be able to go to the DF login screen).

The quickest fix I found is to just change it to the same as RHEL 7 (port 9000) and it works. tho I suppose one could try and find the correct unix socket.

Reply:
Nevermind this is fixed, so RHEL8s php-fpm seems to use unix instead of an ip by default. I changed the nginx script to reflext this and it all works fine now